### PR TITLE
Allow for early termination of the visitor

### PR DESCRIPTION
**Pull request by LukasPietzschmann**

### CODE REVIEW

File: R/visit.R
Comment (EagleoutIce) [inline R/visit.R:4]: Maybe extend this to "anything non-false"? Or do we want a check to enforce `\(n) { magic(n); return TRUE }`

**Review by EagleoutIce — APPROVED** [review]

File: R/visit.R
Comment (LukasPietzschmann) [reply R/visit.R]: Nah, I don't like requiring an explicit return to indicate default behavior. I fixed the doc comment.

**Comment by EagleoutIce** [comment]
I am fine with this! You as well?

**Comment by LukasPietzschmann** [comment]
> You as well?

Who is _you_? Me?

**Comment by EagleoutIce** [comment]
Yes :laughing:

**Comment by LukasPietzschmann** [comment]
Ah lol. Yeah, with bd28342 I'm fine. Aren't we waiting for @Ellpeck's approval?

**Comment by EagleoutIce** [comment]
Yes :)

**Review by Ellpeck — APPROVED** [review]

**Pull request merged**
